### PR TITLE
[feat]:support only sequence feature && fix neg sampler  bug for sequence feature

### PR DESCRIPTION
**Pull request by lgqfhwy**

### CODE REVIEW

File: easy_rec/python/layers/input_layer.py [hunk 1]
```diff
@@ -68,13 +68,18 @@ def __init__(self,
   def has_group(self, group_name):
     return group_name in self._feature_groups
 
-  def target_attention(self, dnn_config, deep_fea, name):
+  def target_attention(self,
+                       dnn_config,
+                       deep_fea,
+                       name,
+                       only_sequence_feature=False):
```
Comment (chengmengli06) [inline easy_rec/python/layers/input_layer.py:75]: what does this params mean?

Comment (chengmengli06) [inline easy_rec/python/layers/input_layer.py:75]: what does only_sequence_feature mean?

File: easy_rec/python/layers/input_layer.py [hunk 2]
```diff
@@ -68,13 +68,18 @@ def __init__(self,
   def has_group(self, group_name):
     return group_name in self._feature_groups
 
-  def target_attention(self, dnn_config, deep_fea, name):
+  def target_attention(self,
```
Comment (chengmengli06) [inline easy_rec/python/layers/input_layer.py:71]: it is better to refactor this code by move target_attention into a separate file named like:  target_attention_layer.py

**Review by chengmengli06 — CHANGES_REQUESTED** [review]

File: easy_rec/python/layers/input_layer.py (hunk 1 above)
Comment (lgqfhwy) [reply easy_rec/python/layers/input_layer.py:75]: 只返回过完 target_attention 之后的特征，不再和 key 进行 concat 了。

Comment (lgqfhwy) [reply easy_rec/python/layers/input_layer.py:75]: 同上。

File: easy_rec/python/layers/input_layer.py (hunk 2 above)
Comment (lgqfhwy) [reply easy_rec/python/layers/input_layer.py:71]: 下一个 pr 这样搞，同时重构一下，然后还支持 key 和 hist_seq 不相等的情况。

File: docs/source/feature/feature.rst [hunk 3]
```diff
@@ -270,6 +270,7 @@ Sequense类特征格式一般为“XX\|XX\|XX”，如用户行为序列特征
     sequence_features: {
       group_name: "seq_fea"
       allow_key_search: true
+      only_sequence_feature:false
```
[the remark this answers is not in the document]
Comment (chengmengli06) [reply docs/source/feature/feature.rst:273]: need_query_feature

**Review by chengmengli06 — APPROVED** [review]

**Pull request merged**